launchcontainer/launchcontainer.py

```diff
@@ -106,26 +96,51 @@ class LaunchContainerXBlock(XBlock):
 
     @property
     def wharf_url(self, force=False):
-        # TODO: logger.debug the failed validations.
         site_wharf_url = None
         if siteconfig_helpers:
             site_wharf_url = siteconfig_helpers.get_value('LAUNCHCONTAINER_WHARF_URL')
+
         urls = (
             # A SiteConfig object: this is the preferred implementation.
-            site_wharf_url,
-            # TODO: Maybe we can set up a signal to update this value if
-            # a SiteConfig object is changed.
+            (
+                'SiteConfig', site_wharf_url
+            ),
+            # TODO: Maybe we can cache this and set up a signal
+            # to update this value if a SiteConfig object is changed.
             # A string: the currently supported implementation.
-            settings.ENV_TOKENS.get('LAUNCHCONTAINER_WHARF_URL'),
+            (
+                "ENV_TOKENS['LAUNCHCONTAINER_WHARF_URL']",
```

Comment:
The values in the site config and in the env vars are simply strings, but I changed this to a tuple to make the logs more transparent.